hibernate-reactive-core/src/main/java/org/hibernate/reactive/persister/entity/impl/ReactiveAbstractPersisterDelegate.java

```diff
@@ -45,21 +42,30 @@ public class ReactiveAbstractPersisterDelegate {
 	private final ReactiveSingleIdEntityLoader<Object> singleIdEntityLoader;
 	private final ReactiveMultiIdEntityLoader<?> multiIdEntityLoader;
 
+	private final ReactiveNaturalIdLoader<?> singleNaturalIdLoader;
+
+//	private final ReactiveMultiNaturalIdLoader<?> multiNaturalIdLoader;
```

Comment:
Please, don't add comments like this. We haven't implemented this yet, just remove it